ang/civicase/ContactCaseTab.html

```diff
@@ -0,0 +1,38 @@
+
+<div id="bootstrap-theme" class="clearfix civicase__container">
+  <h1 crm-page-title class="element-invisible">Related cases to the user</h1>  <!-- Added for SEO purpose (This will be hidden through css) -->
+  <div class="civicase__contact-cases clearfix">
+    <a ng-if="checkPerm('add cases')"
+      class="btn-primary btn civicase__contact-cases-add"
+      ng-href="{{ 'civicrm/case/add' | civicaseCrmUrl:{reset: 1, action: 'add', context: 'standalone'} }}"
+      crm-popup-form-success="refresh()">
+      <i class="material-icons">add_circle</i>
+      {{ ts('Add case') }}
+    </a>
+    <div class="clearfix" ng-if="totalCount > 0">
+      <div class="civicase__contact-cases-container pull-left col-md-6">
+        <div  
+          ng-repeat="case in casesList" 
+          class="panel panel-default civicase__contact-cases-list">
+          <div class="panel-heading">
+            <h3 class="panel-title"> {{ case.title }}</h3>
+          </div>
+          <div class="panel-body panel-body-extra nopadding">
```

Comment:
Whats `nopadding`?

Reply:
Why is `panel-body-extra` required?

Reply:
I have added `nopadding` to test something. Removing it as it is irrelevant
Also `panel-body-extra` was used to accommodate some styles but just checked that removing this doesn't affect the UI so removing this as well.